### DISCUSSION TITLE
[Feature]: exclude/include folders and all sub-folders

### DISCUSSION THREAD

**Discussion by Robin-Sch** [immich-app/immich#1620] [Feature Request]
### Feature detail

Let's say I have the following folders containing pictures:
DCIM/x
DCIM/y
DCIM/z

Right now they all show up individually when selecting albums to backup. Album x, album y, and album z.

It would be useful if I just could select the folder DCIM, and it would upload all images in all sub-folders. Instead of having to individually select album x, album y and album z.
![example](https://user-images.githubusercontent.com/46354497/211201977-1d61273f-f48f-47eb-a485-d0062427c761.png)

At the same time, when someone has a lot of individual folders with a few images, this would make the list of albums to select from huge. It would be nice if you could select a folder and exclude it from showing up in that album picker list.
To further improve the album picker it could be a grid view instead of a slider/list view. Because if you have many albums and you need to select the last one you need to scroll a bit before you reach it. It's not a deal breaker, but again, would be nice.

### Platform

Mobile App

**Comment by Robin-Sch** [comment]
I've updated the example and hopefully it makes (more) sense now?

**Comment by boced66** [comment]
Hello, 

I have the same request, 
When I take a burst photo, my phone creates subfolders for each burst, 
The sub-folders sound themselves in a sub-folder “Cshot”. 

I would like to be able to synchronize the subfolders without having to select each burst, 

I see that the request is over a year old, is there anything planned? 

Thanks for your help,

**Comment by hkp2k** [comment]
Same request here.  Pixel phones/cameras have a portrait feature that creates a new subfolder for each shot.  Synchronizing these photos would be a huge benefit.

**Comment by bo0tzz** [comment]
#5148
